Updating Croatian translation

COMBOhrenovke opened the pull request.

COMBOhrenovke committed:
Updated Croatian translation

COMBOhrenovke committed:
Delete hr_hr.json

COMBOhrenovke committed:
Add files via upload

COMBOhrenovke committed:
Rename en_us.json to hr_hr.json

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/thexaero/Better-PVP-Translations?pullRequest=2) <br/>All committers have signed the CLA.

thexaero commented:
Thank you for your contribution! I know it's not easy to translate some of my weird wording, so great job 👍 
I have read your PM on CurseForge, and yeah, you need a .lang file. I think you can convert it pretty quickly with a few "Replace All"'s in a text editor. Don't forget to remove the escape character \ too. I'll review the translations themselves after that.

COMBOhrenovke committed:
Add files via upload

COMBOhrenovke committed:
Delete hr_hr.lang

COMBOhrenovke committed:
Delete hr_hr.json

COMBOhrenovke committed:
Rename en_us.json to hr_hr.lang

COMBOhrenovke committed:
Delete hr_hr.lang

COMBOhrenovke committed:
Add files via upload

COMBOhrenovke commented:
Okay, thanks! I am crossing my fingers that will work fine now. I had to delete og hr_hr.lang to add the new one, and also remove the hr_hr.json I made, so that's why there are a lot of changes. The process of making a lang out of json isn't hard at all and, as you said, it was just two or three "Replace All"s and then renaming the file. Sometimes, Croatian specific characters like č, ć, š, đ and ž  display wrongly in game, so please check before making an update to mods. It is easily fixed by forcing files to use UTF-8 and nothing else. And once again thank you for all the help in translating. If you weren't so helpful, I would probably give up. Also, btw, why do you use lang files when on newer versions you need json formatted files, and how do you convert them, automatically or manually? Anyways, see you and goodbye!

thexaero commented:
Great, thanks!
And no problem. I use lang files because I still support older MC versions and have an automatic lang to json converter. Might make a json to lang one in the future and switch to json here.
Ok, so I instantly noticed an issue with the lang file. You removed all quote characters instead of removing just the backslashes. Probably easier to reconvert it from json again rather than reinserting them back. 
Cool to see that you've improved a lot of the current lines too :)

COMBOhrenovke commented:
Yeah, in commits, there should still be the old .json file. I kinda goofed big time by removing them, but it should not be hard to re-add them either by searching for places to add them to or by making a new .lang from .json. I'll try to find them manually, and if that does not work out, I'll try the second method.  Finding them won't be hard as in Croatian auto is automatsko, and so on. I will edit the existing one on my fork of translations, and then you can double check if I forgot something. Hope there will be no issues, and yes I improved current ones because some were not like in the rest of the game (for instance using different synonyms or even completely different words), dialectal words or grammatically incorrect. None of this were like game breaking or unreadable, but this new ones are just a bit better ;)

COMBOhrenovke commented:
And also, most of the words that have been  "de-quoted" have a big first letter, so they are real easy to find.

COMBOhrenovke committed:
Update hr_hr.lang

Fix for some quotes that were accidentally removed in the process of making a .lang out of .json file.

COMBOhrenovke commented:
So, as I was doing this, I saw that two of the quotes don't make any sense in Croatian so I actually didn't re-add them. The rest of them I did re-add, and actually found that I accidentally translated sneaking as sprinting at one point XD

thexaero commented:
Great!
Doesn't seem like you removed any of the pre-existing quotes though. So you probably removed some quotes that you have added yourself. I'll try to review the translations soon. I'm busy with 1.16.2 atm, but I'll try to get to it ASAP.

COMBOhrenovke commented:
Okay, I understand that, I am also working on a mod, but it is currently still in works (like really early in development). I want to make it for the newest version, and mc updates ruin my job. Anyways, this translation is not like top priority thing, so take your time with 1.16.2 and when you review translations, write opinions to me, so we can chat about them. The quotes that I deleted were in original, I didn't add any new quotes. I basically deleted only ones around auto because they sounded really off.

thexaero commented:
Ok, I misunderstood you. I thought you meant the quotes were in the current translation used by the mod. I get it now though.
Best of luck with your mod! 
1.16.2 is a minor update but Mojang added experimental custom biomes, which is pretty huge and does affect my mods. Going to move to string ids for biomes for the 1.16.2 port. Although integer ids are still used in world saves, I have a feeling they're going to abandon them very soon, likely in 1.17, so it's better to switch now. Otherwise 1.16.2 -> 1.17 multiplayer map data conversion might not support custom biomes.

COMBOhrenovke commented:
Man thanks for the support! I'll for sure remind you when I launch it, for you to check it out! And also, last year they secretly released support for 3D biomes in 1.15 snapshots and used them in Nether Update for adding multiple biomes on top of each other. I think that not only that they will be removing int id system for biomes in future updates (1.17 or 1.18) but they maybe even add additional nether, end or cave biomes, cause now they can do that with 3D biomes. It is kind of weird that they in game use string ids like minecraft:birch_forest or something and then in save files numbers. Custom biomes seem cool on surface, but they lack new content, which modded ones have. Anyways, biome systems have been changing and mojang is probably nowhere near done with them, so modders will have issues for quite a while in the future until they finalize changes. It is cool that you are going to try and make the change before even them to avoid issues in the future and a really good move on your part. I wish that your work pays off in the long run and that you won't have major issues in doing that as (at least to me) it seem hard to do that stuff. 😄

thexaero commented:
Yeah, there was no way for me to miss 3D biomes tbh since I'm making map mods haha. And yeah, custom biomes may not be as custom as people would like but they still exist. And mod API's will likely build on top of that system anyway.
I'm doing it early because of the experience I have had with them abandoning blockstate ids in the world saves. Also dimension ids. It wasn't a huge deal with blockstate and dimension ids because people don't upgrade modded worlds to new Minecraft versions that often, and I haven't seen any complaints. With biomes, it might affect vanilla worlds, so it's important to do it before they drop int ids.

thexaero commented on hr_hr.lang:
```diff
@@ -1,215 +1,349 @@
+gui.xaero_not_ingame=Moraš biti u igri za korištenje načina uređivanja!
+gui.xaero_welcome_edit_mode=Dobrodošli u način uređivanja!
+gui.xaero_howto_select=Lijevi klik na nekom od sučelja za označavanje.
+gui.xaero_howto_drag=Lijevi klik i povlačenje na odabranom sučelju za pomicanje tog sučelja.
```
Translates back as "Left click and drag on the selected interface to move that interface", which doesn't seem entirely correct. The original says "Left click and drag to move an interface around", which doesn't mention selected interfaces. Please make sure this is correct.

COMBOhrenovke replied:
This is correct, but as it is really long I can change it back to literal translation of your string "Left click and drag to move an interface around", which would be "Lijevi klik i povlačenje za pomicanje sučelja naokolo".

COMBOhrenovke replied:
I later did what I said I would do and reverted to literal translation aka "Lijevi klik i povlačenje za pomicanje sučelja naokolo".

thexaero replied:
Thanks!

thexaero commented on hr_hr.lang:
```diff
@@ -1,215 +1,349 @@
+gui.xaero_box_armour_status=Sučelje koje prikazuje oklop koji trenutno nosiš i predmet u ruci.
+gui.xaero_box_sprint=Trčanje držanjem samo jedne tipke (podesivo).
+gui.xaero_box_sneak=Omogućuje uključivanje/isključivanje šuljanja i stajanje šuljajući se bez potrebe držanja tipke.
+gui.xaero_box_notifications=Obavijesti na zaslonu koje prikazuju različite stvari kao što su: niska glad, nisko zdravlje, eksplozija u blizini ili kad si pogođen strijelom/ama.
```
The old translation seems more correct about the arrow. I know it says "shot by an arrow" in the original, but you don't actually have to get hit for the notification to trigger.

COMBOhrenovke replied:
As I later explained, this was done because correct translation sounds like something from some of Shakespeare's books, aka very weird. You can choose from some of those I typed down there or we can revert it to the old translation.

thexaero replied:
I'm not saying that you should revert to the old translation, but your translation just doesn't seem correct. Is it possible to fix it without sounding weird?

COMBOhrenovke replied:
I could change it to "...ili kad si promašen ili pogođen strijelom/ama". It is in line with the third idea for translation down there. IMO this is the best but longest translation that would be correct.

*note: strijelom is singular, strijelama is plural. I didn't wan't to type them separately as text is already really long and as this is still correct in Croatian.

thexaero replied:
Don't worry about the text being long in the tooltip. There is plenty of space. The translation you suggested seems great!

thexaero commented on hr_hr.lang:
```diff
@@ -1,215 +1,349 @@
+gui.xaero_box_sneak=Omogućuje uključivanje/isključivanje šuljanja i stajanje šuljajući se bez potrebe držanja tipke.
+gui.xaero_box_notifications=Obavijesti na zaslonu koje prikazuju različite stvari kao što su: niska glad, nisko zdravlje, eksplozija u blizini ili kad si pogođen strijelom/ama.
+gui.xaero_box_xp=Bodovi iskustva koje si dobio ili izgubio lete preko tvog zaslona.
+gui.xaero_box_quick_use=Sistem tipki sličan akcijskoj traci koji ti dopušta automatsko korištenje određenog predmeta u svojoj vrućoj traci bez potrebe za pomicanjem sa svojeg trenutno držanog predmeta. Pritiskanje određene tipke omogućuje ti da nastaviš s korištenjem predmeta kao da si pritiskao desni gumb na mišu. Npr.: postavljanje baklji, TNT-a, bacanje napitaka, jedenje hrane, pijenje.
```
"Pritiskanje određene tipke omogućuje ti da nastaviš s korištenjem predmeta kao da si pritiskao desni gumb na mišu" 
Does this mean "Holding the key bind lets you keep using the item like if you were holding the right mouse button"?
In the sense of pressing it for a long time and not a quick press/release.

COMBOhrenovke replied:
I thought at the time that "Pritiskanje" wold be fine, but I'll change it into "Držanje". Because "Držanje" is like holding a object, example a baby, and not a button on keyboard, It was more natural to have "Pritiskanje" as it means pushing down on something for a long time, example being a key on keyboard or piano. I get why you think that it could be confusing for people, but in game, that is how it is translated. Changing it here would add another word meaning the same and that would be bad. Anyways, I'll change it.

thexaero replied:
You know better than me, so if you feel like Pritiskanje works better, then you should keep it. "Pushing down on something for a long time" is exactly what we need.

thexaero commented on hr_hr.lang:
```diff
@@ -1,215 +1,349 @@
+gui.xaero_hunger_setting=Obavijest o gladi
+gui.xaero_hp_setting=Obavijest o zdravlju
+gui.xaero_explosion_setting=Obavijest o eksploziji
+gui.xaero_being_shot_setting="Pogođen si" obavijest
```
Again, "Being Shot" doesn't necessarily mean "Being Hit", if this is what it means here.

COMBOhrenovke replied:
Reason why I didn't translate it completely correct is that "Being shot" could be translated in two ways meaning the same, and one extra not meaning Being shot but closest to what in game messages tell.
1. "Biti pucan" - literal translation that sounds really off
2. "Nešto te puca" - Something is shooting you
3. "Pogođen si" - You have been hit or You're hit

thexaero replied:
Can it be translated as "either hit or missed by an arrow"?

COMBOhrenovke replied:
Yes it can be, but it would be long.
Promašen si ili pogođen
I tried checking with Google Translator, and he said bunch of nonsense, so ignore it on the text above
We can either do one of these three things:
1. Revert to old translation
2. Use "Something is shooting you" which in Croatian doesn't necessarily mean hitting but just shooting
3. Use  "Hit or missed by an arrow" translation from above

COMBOhrenovke replied:
I'll use the same as the above solution, aka "Promašen si ili pogođen" obavijest. This issue is also fixed now.

thexaero commented on hr_hr.lang:
```diff
@@ -1,215 +1,349 @@
+gui.xaero_black=Crna
+gui.xaero_dark_blue=Tamno plava
+gui.xaero_dark_green=Tamno zelena
+gui.xaero_dark_aqua=Tamno tirkizna
```
Aqua and turquoise are actually different colors, you can look up the difference. Is there no word for aqua? Please let me know.

COMBOhrenovke replied:
This was made by using in game translations and they sound off to me too but that's how it is. Aqua in Croatian is translated as akva but still written as aqua. Because aqua uses q and q isn't in Croatian alphabet, people do say word aqua but never write it, and instead type something close to it. Turquoise is closest to aqua so aqua is called turquoise meaning "tirkizna" as I wrote and as it stands in game. Idk why it is like that but, even though it is stupid it is how it needs to be, so I won't change it. TLDR there is no aqua and turquoise is closest to aqua, and I know they're not the same.

thexaero commented on hr_hr.lang:
```diff
@@ -1,215 +1,349 @@
+gui.xaero_dark_gray=Tamno siva
+gui.xaero_blue=Plava
+gui.xaero_green=Zelena
+gui.xaero_aqua=Tirkizna
```
Same as the other.

COMBOhrenovke replied:
Same as the other.

thexaero commented on hr_hr.lang:
```diff
@@ -1,215 +1,349 @@
+gui.xaero_slime_chunks=Dijelovi svijeta s ljigavcima
+gui.xaero_unlimited=Neograničeno
+gui.xaero_old_deathpoints=Zadrži putne točke stare smrti
+gui.xaero_toggle_map=Uključi mini kartu
```
Does "Prebaci" work here?

COMBOhrenovke replied:
Yes, it would work fine, I'll change it. Prebaci was taken from in game translation for toggle in sprint and sneak settings under accessibility. I added Uključi as I thought that Prebaci mini kartu could be confusing for people because it can mean 3 things. Literal translations are:
- Uključi - turn on
- Prebaci - move, toggle, switch

COMBOhrenovke replied:
I later decided not to put Prebaci, so text wasn't changed.

thexaero replied:
Ok, got it. But isn't "turn on" incorrect? It also turns it off.
You can try something like "turn on/turn off", since there's a bit more space in the controls menu I think.

COMBOhrenovke replied:
No it isn't. Most of the time when you could type Turn on/off, we type just Turn on bcz it saves space and it implies that if it isn't turned on it is turned off. Translation for "Turn on/off" or "Turn on/Turn off" is "Uključi/Isključi". Many, and I mean it really many guis in the game say just Uključi, but this is still an option if you really want it. Both of them are equally correct.

thexaero replied:
Alright, then it's good as is!

thexaero commented on hr_hr.lang:
```diff
@@ -1,215 +1,349 @@
+gui.xaero_vanilla=Vanilla
+gui.xaero_lighting=Munja
+gui.xaero_air_setting=Obavijest o niskom zraku
+gui.xaero_air_low=Zrak nizak
```
"Niski zrak" seems more correct to me. Does your translation mean "low air"? 
"The air is low" isn't correct, if that is what this means. This setting is for setting what is considered "low air". Apologies if I'm wrong!

COMBOhrenovke replied:
Oh, I understood it wrongly. Yes it should be Niski zrak in that case. "Zrak nizak" means "Air low". I thought it has something to with air bubbles and drowning.

thexaero replied:
Thanks! It does have to do with air bubbles, but it's a setting though, not a notification.

thexaero commented on hr_hr.lang:
```diff
@@ -1,215 +1,349 @@
+gui.xaero_lighting=Munja
+gui.xaero_air_setting=Obavijest o niskom zraku
+gui.xaero_air_low=Zrak nizak
+gui.xaero_looking_at=Gledaš prema
```
Please check this too. Not sure if it's correct. The option value is for the _gui.xaero_show_distance=Distance To WP_ option and means only displaying distance for waypoints that you are "looking at".

COMBOhrenovke replied:
"Koju gledaš" as  gui.xaero_show_distance=Distance To WP is "Udaljenost od PT" if I am not wrong.
Koju gledaš means "which you are looking at".

thexaero replied:
Sounds perfect! Thanks.

thexaero commented on hr_hr.lang:
```diff
@@ -1,215 +1,349 @@
+gui.xaero_toggle_waypoints=Uključi putne točke u igri
+gui.xaero_dots_scale=Skala točaka entiteta
+gui.xaero_entity_info=Informacije o entitetu
+gui.xaero_toggle_slime=Uključi dijelove svijeta s ljigavcima
+gui.xaero_toggle_grid=Uključi mrežu dijelova svijeta
```
Prebaci?

COMBOhrenovke replied:
Nope, here it would sound silly.

thexaero commented on hr_hr.lang:
```diff
@@ -1,215 +1,349 @@
+gui.xaero_delete_set=Obriši set
+gui.xaero_delete_set_message=Jesi li siguran želiš li obrisati trenutni set?
+gui.xaero_delete_set_message2=Set će biti obrisan iz trenutnog svijeta.
+gui.xaero_toggle_waypoints=Uključi putne točke u igri
```
Prebaci?

COMBOhrenovke replied:
Nope, here it would sound silly.

thexaero replied:
"Turn on/turn off"? Maybe even shorten it somehow, if possible.

COMBOhrenovke replied:
Same as above, wouldn't change it. Uključi is correct and far more used than Uključi/Isključi.

thexaero commented on hr_hr.lang:
```diff
@@ -1,215 +1,349 @@
+gui.xaero_show_light_level=Prikaži razinu svjetline
+gui.xaero_up=[Idi gore]
+gui.xaero_down=[Idi dolje]
+gui.xaero_display_all_sets=Uključi prikaz PT setova
```
Prebaci?
Also, please check if this means "display ALL sets", not just "display sets". Usually only 1 set is rendered at a time, this toggles rendering all sets.

COMBOhrenovke replied:
Nope, here it would sound silly. Corrected version would be "Uključi prikaz svih setova PT". Svih=All

thexaero replied:
Thank you!

thexaero commented on hr_hr.lang:
```diff
@@ -1,215 +1,349 @@
+gui.xaero_box_entity_depth=Prikaži entitete tamnije ovisno o njihovoj Y razini u odnosu na tebe.
+gui.xaero_box_height_limit=Maksimalni razmak Y razina za entitete dok se više ne prikazuju.
+gui.xaero_box_compass_over_wp=Prikaži slova na kompasu (S, I, J, Z) preko putnih točaka na karti.
+gui.xaero_toggle_map_waypoints=Uključi putne točke na karti
```
Prebaci?

COMBOhrenovke replied:
Nope, here it would sound silly.

thexaero commented on hr_hr.lang:
```diff
@@ -1,215 +1,349 @@
+gui.xaero_team=Tim
+gui.xaero_world_server=Svijet/Poslužitelj
+gui.xaero_subworld_dimension=Podsvijet/Dimenzija
+gui.xaero_switch_waypoint_set=Zamjeni set putnih točaka
```
Please check if this is correct. "Zamjeni" seems weird to me.

COMBOhrenovke replied:
No, sorry but Zamjeni is okay because it means Replace but in way of switching. It is kinda hard to explain but it works like this so...

thexaero replied:
How about "promijeniti"?

COMBOhrenovke replied:
Promjeniti would mean change out like Zamjeniti does. Translation with Promjeniti would go like this:
"Promjeni set putnih točaka"
I would still go with Zamjeni as it is more natural to me, but this is an option. I really appreciate you searching for alternative words to use, very cool!

thexaero replied:
Go with the one you like more! And wait. Is it supposed to be spelt as "Promjeniti" without the i? Why does Google add the i?

COMBOhrenovke replied:
So, I checked, both Promijeniti, and Promjeni are correct. I goofed up there by removing i in Promijeniti, but that form of the verb won't be used so it doesn't matter that I made a mistake. It is more to show how much with no reason Croatian is hard. Basically, a long time ago Slavic languages had something called jat, a pretty weird sound. As time went, different languages and even dialects would go on to say jat differently. Eventually nobody knew what the letter was meaning, so people typed what they would say instead of jat (basically pronunciations of letter). There are four main pronunciations:
- ije
- je
- e
- i
And unlike bigger half of Slavic languages (they have either one, two or three), Croatian has all of the above mixed in. Many times, different states and forms of the same word have different pronunciations. Google kinda guesses them, but people also get confused about them. Something like saying that milk is typed mljeko instead of mlijeko could grant you a fabulous award of an f- grade. Anyways, I'll leave Zamjeni here and resolve this issue.

thexaero replied:
Alright, thanks for the info haha!

thexaero commented on hr_hr.lang:
```diff
@@ -1,215 +1,349 @@
+gui.xaero_hide_world_names=Sakrij imena svijetova/IP-eve
+gui.xaero_open_slime=Otvori na pritisak tipke
+gui.xaero_options=Opcije
+gui.xaero_make_automatic=Napravi svijet/poslužitelj automatski
```
I don't think this is correct. "Make" in this case should mean "Make something become". For example "make it green", "make it bigger". If your translation means "create", then please fix it. Apologies if I'm wrong.

COMBOhrenovke replied:
I am not angry at your mistake, you needed to check, and apologies accepted. Napravi in this case means "make it green", that is, it stands for changing the state of something not creating something new. "Make World/Server Auto" is translated completely correctly.

thexaero replied:
Got it, thanks!

thexaero commented on hr_hr.lang:
```diff
@@ -1,215 +1,349 @@
+gui.xaero_make_automatic=Napravi svijet/poslužitelj automatski
+gui.xaero_make_automatic_msg1=Jesi li siguran?
+gui.xaero_make_automatic_msg2=Ovo će zamijeniti sve podatke o putnim točkama za odabrani svijet/poslužitelj i automatski svijet/poslužitelj, na taj način čineći odabrani svijet automatskim. Budite sigurni da znate što radite.
+gui.xaero_make_multi_automatic=Napravi podsvijet automatski
```
Same as previous.

COMBOhrenovke replied:
Same as previous.

thexaero commented on hr_hr.lang:
```diff
@@ -1,215 +1,349 @@
+gui.xaero_always_show_distance=Uvijek prikazuj udaljenost od PT.
+gui.xaero_item_tooltip=Opis predmeta
+gui.xaero_entity_tooltip_settings=Postavke opisa predmeta
+gui.xaero_item_tooltip_min_lines=Minimum linija
```
Please check if this means "minimum number of text rows".

COMBOhrenovke replied:
Your original text is "Minimum Lines", and my translation is correct, but if it were to be  "minimum number of text rows", I would put "minimalni broj redova teksta", but because it isn't, this is ok as it is now.

thexaero replied:
But does "linija" mean a text row too?

COMBOhrenovke replied:
Yes, linija is both a line and a text row. Row and line are synonyms both in English and Croatian (ofc in context of some kind of text). If I was to translate text row, I could use either linija teksta (shortened to just "linija") or  red teksta. Full sentences would sound like this:
1. Minimum linija (teksta)
2. Minimum redova (teksta)
*teksta is in brackets because it can be skipped (I would skip it), but if you really want to have text in this string, teksta is the correct form of the word to use.

COMBOhrenovke replied:
If we are translating from your original, I still think that my translation is completely fine. I don't understand why you asked if linija is also a text row, but it made me laugh!

thexaero replied:
It's just that Croatian seems VERY similar to Russian and linija does not work for text rows in Russian, only for the other meaning of line. "Linija teksta" is **exactly** how you would say it in Russian, if you could use that word. But if it works, then this is great!

COMBOhrenovke replied:
Hahaha, all Slavic languages are kinda interconnected. I didn't know that Russian people would also say it like that, and Croatian people hardly understand Russian, so I wouldn't say they are similar. Russians speak very fast (partly because of the alcohol) and to me at least, they are like animal sounds, bunch of noise that doesn't make any sense, for say a cat. Also Russian uses Cyrillic script while Croatian uses Latin script. We did use all three scripts (Croatian angular Glagolitic script, Gajica Latin script, Bosančica Cyrillic script) for a long time but now we don't. Croatia is also far away from Russia so our language is more similar to Bosnian, Serbian and Montenegrin, yet being distinct from them in many ways. If I were to talk to some of them, I would probably understand the most, but some words I would ask for explanation and then go on with talking. Btw for your own sake, please don't tell something like this to other Slavic people, because they would probably try to commit crime on you. I am okay with this but many people aren't.

thexaero commented on hr_hr.lang:
```diff
@@ -1,215 +1,349 @@
+gui.xaero_temporary=privremeno
+gui.xaero_restore=Vrati
+gui.xaero_biomes_vanilla=Biomi u Vanilla načinu
+gui.xaero_potion_effects_blink=Trepni kad si nizak
```
Translates back as "Blink when you are low". Does this work for potion effects being low?

COMBOhrenovke replied:
Kind of. Did your text mean "Blink (in sense of some particles) when potion effects are low". In that case I would say that this is slightly wrong (albeit still understandable), but as there isn't any shorter translation for this, I would leave it as it is.

thexaero replied:
It should mean that the on-screen potion effect icons are blinking. Does it work for that?

COMBOhrenovke replied:
Hahaha no, I goofed. Trepni would be like blinking with eyes or some kind of light blinking.
I would say that "Zasvijetli" could be good. Sentence would go like:
"Zasvijetli kad je nizak/su niski"
Meaning: Light up when low.
It isn't blinking, but should still work out fine.
I would personally prefer this translation.

thexaero replied:
I don't know if you goofed! Might have been correct.
"Light up when low" wouldn't work because it goes from opaque to transparent to opaque when it blinks. Any way to describe that?

COMBOhrenovke replied:
Trepni, as I said, is like having a light source turn on and of really fast aka blinking, but I wouldn't say that it works for what you described. Is the transition fast or does it take time to go from opaque to transparent? If it is fast, maybe I could use "Svjetlucaj kad je nizak/su niski" ofc referring to potion effect/s running out. Other ideas are "Zatreperi...", or just "Treperi..." . The translation I got from Google is this: Trepnite kad je nizak.
It is very wrong, translates back to English as Blink when he is low/short, Even though Google thinks this is fine, rest assured that it was funny to see for me. I would go with Treperi in this case, but I need to know the answer to the question from above.

thexaero replied:
It goes instantly from one state to another. The blinking rate isn't very high though.

COMBOhrenovke replied:
Okay, then I will use: "Treperi kad je nizak/su niski". Also, for some strange reason, it seems that I have sent previous message twice. It wasn't intended and is probably some kind of github related bug.

thexaero commented on hr_hr.lang:
```diff
@@ -1,215 +1,349 @@
+gui.xaero_waypoint_distance_vertical_visibility_angle=PT udalj. vertik. kut
+gui.xaero_teleportation=Teleportacija
+gui.xaero_teleportation_tooltip=Kad je isključeno, može biti uključeno jedino kroz config dokument!
+gui.xaero_use_multiworld_tooltip=Isključi ako je ovo jednostavan poslužitelj sa jednim svijetom (bez lobia, svijetova za odabir mini igara). Višesvijetna detekcija može samo izazvati probleme na takvim poslužiteljima.
```
Please update to the current line version:
gui.xaero_use_multiworld_tooltip=Disable if this is a simple server with a single world (no lobbies, game mode worlds etc). Multiworld detection can only cause issues on such servers. Installing the mod on server side should prevent the issues though.

COMBOhrenovke replied:
Okay, I am missing only the last sentence, so it shouldn't be a problem.

thexaero replied:
Seems perfect, thank you!

thexaero commented on hr_hr.lang:
```diff
@@ -1,215 +1,349 @@
+gui.xaero_use_multiworld_tooltip=Isključi ako je ovo jednostavan poslužitelj sa jednim svijetom (bez lobia, svijetova za odabir mini igara). Višesvijetna detekcija može samo izazvati probleme na takvim poslužiteljima.
+gui.xaero_world_teleport_command=Razgovorna naredba teleportiraj
+gui.xaero_use_default=Koristi zadano
+gui.xaero_centered_enlarged=Centriraj kad je povećano
+gui.xaero_zoomed_out_enlarged=Odzumiraj kad je povećano
```
Check if "povećano" means "make size bigger" and not "zoom in". This word is used in other lines too. Apologies if it's already correct.

COMBOhrenovke replied:
Zoomed in and povećano are synonyms in Croatian so I could'we typed "Odzumiraj kad je zumirano" which sounds weird, so  I put povećano and it works out fine. Povećano means something that was made bigger and also, magnifying glass is povećalo meaning device that makes things seem bigger. Anyways, this is okay as is.

thexaero replied:
Got it! The correct one is "make size bigger", btw. I assume it works for that?

COMBOhrenovke replied:
Yes, it works perfectly fine!

thexaero commented on hr_hr.lang:
```diff
@@ -1,215 +1,349 @@
+gui.xaero_box_cave_zoom=Koliko zumirati dalje kada je način špiljskih karti uključen.
+gui.xaero_box_entity_depth=Prikaži entitete tamnije ovisno o njihovoj Y razini u odnosu na tebe.
+gui.xaero_box_height_limit=Maksimalni razmak Y razina za entitete dok se više ne prikazuju.
+gui.xaero_box_compass_over_wp=Prikaži slova na kompasu (S, I, J, Z) preko putnih točaka na karti.
```
Umm. I don't think compass letters are translated. Maybe they should be?

COMBOhrenovke replied:
U mean those on the mini map and world map in game?, I think they should be translated, but it's no big deal if you can't do that. I translated them in this string cause I didn't know if they were or weren't translated on the map, so I did it anyways.
N, E, S, W - North, East, South, West
S, I, J, Z - Sjever, Istok, Jug, Zapad

thexaero replied:
I think we can leave it like this. Going to add translation lines for the compass in the future.

thexaero commented on hr_hr.lang:
```diff
@@ -1,215 +1,349 @@
+gui.xaero_teleport_anyway=Svejedno teleportiraj
+gui.xaero_teleport_shows_coordinates=Prikaže ti koordinate!
+gui.xaero_render_all_wp_sets=Prikaži sve setove putnih točaka
+gui.xaero_waypoints_global=Globalno
+gui.xaero_waypoints_local=Lokalno
```
This is used for global/local waypoints, so please make sure this means exactly that. Translates back as "globally" and "locally" atm which doesn't make sense to me. Apologies if this is correct.

COMBOhrenovke replied:
Google translate is a really funny piece of technology. It is correct, and it saying that lokalno means locally made my day. XD

thexaero replied:
I'm trying to translate it to and from multiple languages and it always seems incorrect. 
Does "lokalne" or "lokalna" work? And yeah, I know Google translate can be wrong, but idk.

COMBOhrenovke replied:
Lokalne would be feminine plural, and Lokalna would be feminine singular. If the words before this Global/Local toggle, are Waypoints, then it should be:
1. If singular is used (of waypoints, aka waypoint) - Lokalna/Globalna
2. If plural is used (waypoints) - Lokalne/Globalne
Waypoint aka Putna točka is feminine adjective + noun combo in singular so the later typed adjectives Lokalna/Globalna have to also be feminine singular. Just tell me if it is waypoint or waypoints and I will change out both strings (this one and "Globalno" one.

thexaero replied:
Oh, ok. Damn, Google gets everything wrong lol. It's probably better to use singular. It can mean multiple waypoints, if you select multiple to edit, but usually it's only 1, so let's go with singular.

COMBOhrenovke replied:
Okay, I'll change it!

thexaero commented on hr_hr.lang:
```diff
@@ -1,215 +1,349 @@
+gui.xaero_world_connection_tooltip=Poveži/odkači odabrani podsvijet za/od trenutnog automatskog podsvijeta. Poveži samo podsvijetove koji su iz istog savea svijeta. Na primjer, ako te vanilla Nether portal vodi iz podsvijeta A u podsvijet B, onda ih možeš povezati. Podsvijetovi povezani s tvojim trenutnim su uvijek na vrhu popisa i označeni su sa simbolom *.
+gui.xaero_connect_with_auto_msg=Bi li htijeo stvoriti slijedeću vezu između podsvijetova?
+gui.xaero_disconnect_from_auto_msg=Bi li htijeo obrisati slijedeću vezu između podsvijetova?
+gui.xaero_teleport_not_connected=Pokušavaš se teleportirati do podsvijeta koji nije povezan na trenutni automatski. Ako si siguran da je ovaj podsvijet iz istog pod-poslužitelja/savea svijeta, onda možeš omogućiti teleportaciju dodavajući vezu u izborniku putnih točaka -> Opcije -> Dodaj vezu podsvijetova. Ali, ako povežeš nevezane podsvijetove i teleportiraš se, onda bi se mogao ugušiti u bloku  ...
```
Please add the new lines at the end:
gui.xaero_ignore_heightmaps=Ignore Server Heightmaps
gui.xaero_box_ignore_heightmaps=Some servers put custom values in the world heightmaps, which can cause incorrectly rendered maps. Sometimes it can even happen in heavily modded singleplayer worlds. This option should fix such issues at the expense of a bit of performance. Do not use unless you need to. Reenter the world after toggling the option!
gui.xaero_error_loading_properties=Error loading server minimap properties. Please retry.

COMBOhrenovke replied:
Okay, I will add them in this new version

thexaero replied:
Appreciate it, thank you! Seems perfect.

thexaero reviewed (changes requested):
Phew. This was a long one to go through. Can't believe the old translation was so outdated.
Great job! Most of the translations seem perfect. A couple of lines there are definitely at least questionable, so I would like your input on that, but that's to be expected with this much to translate. 
I also noticed that a few lines don't fit the buttons to the point where it's unreadable. You can test it with this:
[better pvp test jar](https://www.dropbox.com/s/lekm6mcrnybj34r/BetterPvP_20.20.1_Forge_1.12.jar?dl=0)
You can update the test jar yourself by opening it as a _.zip_ archive (WinRAR?) and replacing _assets\xaerobetterpvp\lang\hr_hr.lang_
The lines that only slightly don't fit the buttons are fine. I think I'm going to increase the button size soon.
Again, huge thank you for your contribution! I appreciate your time and work and I'm sure a lot of people will love this.

COMBOhrenovke commented:
I'll start on making the edited version, but will still wait on things that need your approval. And I too am sure that people will be really happy. My friends actually told me about the issues in translation and I said that I will fix them, so they are waiting patiently for me to be done.  Putting time for this wasn't a problem, as I also translate a lot of other things alongside this mod. I hope we are near done with this!

COMBOhrenovke commented:
Also, about button sizes and text out of boxes, you could make it so it shows maximum that it can fit, and just slowly moves text to left when you hover over it and thus allows you to read the rest of the text. Many sites and things have some system similar to this. Another idea is that button text sizes are dynamical in the same way other ui-s work. When you have a bigger screen with bigger resolution, buttons grow in size, and you can easily see the whole text. Idk how hard it would be to implement but it is worth trying out. Another thing I found while playing around in game was that some letters used unicode rather than mojangles font in Minecraft. Idk if those letters weren't added in mojangles at the time of 1.12.2 but it is worth to investigate if such issues occur in later versions up to 1.16.x.  Also, I have talked about this to forge staff but I really like how forge was loading back then and would really like if they didn't change it, but that's just a small thing. Which one do you prefer?

Greg-21 commented:
If I can just add my two penn'orth, I think that a text slowly moving to the left would clearly stand out from the vanilla Minecraft UI design. As for me, the best idea would be to simply increase the button width, make button labels as compact as possible without losing much clarity, and if necessary, put all additional information into a tooltip, which appears when you hover over a button.

This issue seems to be bigger than I thought when I was reporting it here: https://www.curseforge.com/minecraft/mc-mods/xaeros-minimap/issues/204

COMBOhrenovke committed:
Update hr_hr.lang

COMBOhrenovke commented:
> If I can just add my two penn'orth, I think that a text slowly moving to the left would clearly stand out from the vanilla Minecraft UI design. As for me, the best idea would be to simply increase the button width, make button labels as compact as possible without losing much clarity, and if necessary, put all additional information into a tooltip, which appears when you hover over a button.
> 
> This issue seems to be bigger than I thought when I was reporting it here: https://www.curseforge.com/minecraft/mc-mods/xaeros-minimap/issues/204

Yes it would stand out, and it would be a fix for people playing on small resolutions and screens. Most of the time it wouldn't be used as bigger buttons would fix text issues for most of people. And so you know, I don't think that it would stand out. If you are playing with mods you clearly want to add something or change the game, so you shouldn't expect mods to be in line with vanilla. I like my idea so much that now I'll post it on feedback, just to see people's and Mojang's reaction. It would be really helpful, as it would remove the need of thinking about text sizes. Anyways, additional information is already in tooltips and this issue can't be fixed that way as we have a problem with base labels for settings, not "what is this" texts.

thexaero reviewed (commented):
Thank you for the fixes!

thexaero commented:
About the button scrolling/size. Maybe I can give more freedom to the translators when it comes to tooltips. Currently specific options have them, but I can make it be controlled by the translation. This will make it possible to abbreviate the option and put the full name in the tooltip.
And button scrolling as a vanilla MC feature seems like a good idea.

I think we can get this translation out in the update that follows 20.20.2! I hope I don't sound rude in my review comments. I feel like I do, so I apologise. That was not my intention.

thexaero commented:
Some characters looking different is also an issue in vanilla 1.12.2. Not sure if also in newer versions, but that's a Minecraft thing. 
I prefer the old Forge loading screen but Forge mod loading doesn't work the same in 1.13+, so it needed to change. I think they wanted to improve the look of it but then moved on.

COMBOhrenovke commented:
> About the button scrolling/size. Maybe I can give more freedom to the translators when it comes to tooltips. Currently specific options have them, but I can make it be controlled by the translation. This will make it possible to abbreviate the option and put the full name in the tooltip.
> And button scrolling as a vanilla MC feature seems like a good idea.
> 
> I think we can get this translation out in the update that follows 20.20.2! I hope I don't sound rude in my review comments. I feel like I do, so I apologise. That was not my intention.

Thanks, that would help, and I know, button scrolling would be awesome!

COMBOhrenovke commented:
> Some characters looking different is also an issue in vanilla 1.12.2. Not sure if also in newer versions, but that's a Minecraft thing.
> I prefer the old Forge loading screen but Forge mod loading doesn't work the same in 1.13+, so it needed to change. I think they wanted to improve the look of it but then moved on.

I know they needed to almost completely rewrite forge for 1.13 and up, and in the process changed the look. It is really funny that they wanted to improve on current look and never did. I guess it is because of lack of time, but it would still be awesome for them to change the look when they have more time. And I didn't know about that 1.12 issue hahaha. All letters seem fine in 1.16 so it's all good.

COMBOhrenovke committed:
Update hr_hr.lang

COMBOhrenovke commented:
After 105 messages, now we are finally down to one issue!

COMBOhrenovke commented:
https://feedback.minecraft.net/hc/en-us/community/posts/360072665552-Button-scrolling-in-menus
Feedback link for button scrolling by me. I was trying to found if there was something similar but couldn't find anything so I made a  new thread for this. Please go and vote so they see it.

COMBOhrenovke committed:
Update hr_hr.lang

COMBOhrenovke committed:
Merge pull request #1 from thexaero/master

banana

COMBOhrenovke commented:
Fix for the last issue and pull request from your branch to mine so it isn't behind by 3 commits.

thexaero commented:
Thank you so much for your contribution! Everything seems great now, so you can expect it to be included in the next minimap/bpvp update.

COMBOhrenovke commented:
No problem man! You were very helpful the whole time and I am glad I worked on this with you. I hope people love this new translation and also, I plan to update it in case that you add new strings to en_us.lang!

thexaero commented:
I appreciate you being open for discussion!
BTW, I just noticed that some of the newest lines (at the bottom) don't fit the buttons. It's not the biggest deal, but I thought you might want to know. I might add an automatic tooltip system in the future for button names that don't fit. And I noticed a double " mark in the file too. Kind of my fault for missing that earlier. Going to fix it myself. 
Definitely looking forward to your next PR!
